Open Judge System/Workers/OJS.Workers.ExecutionStrategies/JavaUnitTestsExecutionStrategy.cs

```diff
@@ -0,0 +1,319 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+
+    using OJS.Common;
+    using OJS.Common.Extensions;
+    using OJS.Common.Models;
+    using OJS.Workers.Checkers;
+    using OJS.Workers.Common;
+    using OJS.Workers.Compilers;
+    using OJS.Workers.Executors;
+
+    public class JavaUnitTestsExecutionStrategy : JavaZipFileCompileExecuteAndCheckExecutionStrategy
+    {
+        protected const string IncorrectTestFormat =
+            "The problem's tests were not uploaded as an archive of zips. A Lecturer/Administrator should reupload the tests in the correct format.";
```

Comment:
`A Lecturer/Administrator should reupload the tests in the correct format`. IMO `Reupload the tests in the correct format`. is enough. What if some day we have more roles?